src/Mod/Fem/femsolver/calculix/writer.py

```diff
@@ -1160,7 +1160,7 @@ def write_constraints_pressure(self, f):
             f.write("*DLOAD\n")
             for ref_shape in femobj["PressureFaces"]:
                 # the loop is needed for compatibility reason
-                # in depretiated method get_pressure_obj_faces_depreciated
+                # in depreciated method get_pressure_obj_faces_depreciated
```

Comment:
shouldn't this be 'deprecated' ?  
` in **deprecated** method get_pressure_obj_faces_**deprecated**`

Reply:
Looks like there is quite a few mentions in 
```
src/Mod/Fem/femmesh/meshtools.py:# ***** depreciated method for retrieving pressure faces *****************************************
src/Mod/Fem/femmesh/meshtools.py:def get_pressure_obj_faces_depreciated(
src/Mod/Fem/femmesh/meshtools.py:    # see comment on get_pressure_obj_faces_depreciated in the regard of getccxVolumesByFace()
src/Mod/Fem/femsolver/writerbase.py:        # depreciated version
src/Mod/Fem/femsolver/writerbase.py:            femobj["PressureFaces"] = meshtools.get_pressure_obj_faces_depreciated(
src/Mod/Fem/femsolver/writerbase.py:            # the data model is for compatibility reason with depreciated version
src/Mod/Fem/femsolver/writerbase.py:            # get_pressure_obj_faces_depreciated returns the face ids in a tuple per ref_shape
src/Mod/Fem/femsolver/writerbase.py:            # some_string was the reference_shape_element_string in depreciated method
src/Mod/Fem/femsolver/calculix/writer.py:                # in deprecated method get_pressure_obj_faces_depreciated
```

Reply:
Why does this command does not find thjem?

`codespell -q 2 -S *.ts -L childs,dof,dum,methode,nd,normaly,uint,vertexes,freez src/Mod/Fem/`

Reply:
Two reasons:  
1.  I used a different dictionary file from https://github.com/orbitcowboy/codespell_dictionary (though it has many errors in it, it's still useful)  
2. codespell still isn't sophisticated enough to encompass usecases and edgecases that it encounters.